docs/core/deploying/native-aot/index.md

```diff
@@ -14,10 +14,21 @@ The benefit of native AOT is most significant for workloads with a high number o
 
 The native AOT deployment model uses an ahead-of-time compiler to compile IL to native code at the time of publish. Native AOT apps don't use a just-in-time (JIT) compiler when the application runs. Native AOT apps can run in restricted environments where a JIT isn't allowed. Native AOT applications target a specific runtime environment, such as Linux x64 or Windows x64, just like publishing a [self-contained app](../index.md#publish-self-contained).
 
-There are some limitations in the .NET native AOT deployment model, with the main one being that run-time code generation isn't possible. For more information, see [Limitations of Native AOT deployment](#limitations-of-native-aot-deployment). The support in the .NET 7 release is targeted towards console-type applications.
+## Limitations in the .NET native AOT deployment model
+
+### [.NET 7](#tab/net7)
 
 > [!WARNING]
```

Comment:
The warning here is a bit weird. Console apps were fully supported with NativeAOT in .NET 7 - but this styling looks like it's something to avoid.

Reply:
#36529